packages/calling-component-bindings/src/baseSelectors.ts

```diff
@@ -86,6 +86,17 @@ export const getRemoteParticipants = (
       [keys: string]: RemoteParticipantState;
     }
   | undefined => {
+  /* @conditional-compile-remove(rooms) */
+  // Filter out consumers from remote participants
+  {
+    const remoteParticipants = { ...state.calls[props.callId]?.remoteParticipants };
+    Object.keys(remoteParticipants).forEach((k) => {
+      if (remoteParticipants[k].role === 'Consumer') {
+        delete remoteParticipants[k];
```

Comment:
do we want to delete? or just filter out?

Reply:
I created a copy of remote participants from state so it is safe to delete it from the copy to essentially filter out consumers when the remote participants are passed to the composite

Reply:
nit: `filter` would be a better option here for readability (easy to miss that the object is cloned before mutation, so in refactors it could be lost)

Consider:
```ts
return remoteParticipants.filter((remoteParticipants) => remoteParticipants.role !=== 'Consumer');
```